src/test/java/org/cactoos/io/TeeInputStreamTest.java

```diff
@@ -47,7 +47,7 @@
     public void copiesContentByteByByte() throws Exception {
         final ByteArrayOutputStream baos = new ByteArrayOutputStream();
         final String content = "Hello, товарищ!";
-        MatcherAssert.assertThat(
+        new Assertion<>(
             "Can't copy InputStream to OutputStream byte by byte",
```

Comment:
@marceloamadeu can you update the message of the test you modified to follow the same form as explained before?